test/include/materials/VectorCoupledValuesMaterial.h

```diff
@@ -25,14 +25,18 @@ class VectorCoupledValuesMaterial : public Material
 protected:
   virtual void computeQpProperties() override;
 
+  const bool _request_dotdot;
+
   const VectorVariableValue & _value;
+  const ADVectorVariableValue & _ad_value;
   const VectorVariableValue & _dot;
-  const VectorVariableValue & _dot_dot;
+  const VectorVariableValue * _dot_dot;
```

Comment:
Why don't we ever do this?
```
const VectorVariableValue * const _dot_dot;
```
In these cases?

Reply:
Definitely could do that in this case since we're setting it up in the initialization list. Sigh probably should change it since users may look to these for examples

Reply:
Sounds good. I'll create an issue and do a few greps later today to find other cases like this that we can constify.

Reply:
Or if you want we can merge this now and I can include it in the other PR.